app/views/users/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<h3 class="teal-title">User Summary: <%= @site_user.name %></h3>
+<p tiny-black>Joined site <%= @site_user.created_at.strftime("%b %d, %Y")%></p>
+<h4 class="black-subtitle">Votes</h4>
+
+<table>
+	<thead>
+		<tr>
+			<th>Media Title</th>
+			<th>Created By</th>
+			<th>Published</th>
+			<th>Category</th>
+			<th>Voted On</th>
+		</tr>
+	</thead>
+	<tbody>
+		<% @site_user.votes.each do |vote| %>
+			<% work = vote.work %> <!-- QUESTION: too dependent?? -->
+			<tr>
+				<td><%= link_to work.title, work_path(work.id) %></td>
+				<td><%= work.creator %></td>
+				<td><%= work.get_publication_year %></td>
+				<td><%= work.category %></td>
+				<td><%= work.created_at.strftime("%b %d, %Y") %></td>
+			</tr>
+		<% end %>
```

Comment:
In the spirit of POODR and what we've been talking about with dependency,  I felt like this was too dependent but wasn't sure. I'm going through each user's votes to access the works, which definitely feels dependent-y. But I wasn't sure since they have relationships. BUT that also means I'm depending on these relationships to exist, which they may or may not in the future. 

I considered getting the Work attributes as hashes and dealing with the information in the hash. But this seemed a bit excessive and started causing a lot of error, so I tabled this idea due to time constraints and being unsure of this approach in general. I left the commented out code for this approach in the User and Work model just in case.

Is this an example of dependency or poor design that I should rework? If so, any feedback on this would be greatly appreciated!

Reply:
Just took a glance at this:

I wouldn't have seen this code and personally thought "these models have a relationship that's showing tight coupling that concerns me"

That being said, with your proposed solution of making work info available via votes to user is a fun solution; while again I wouldn't necessarily have thought this was needed at this point of the project, I could imagine a realistic project needing something like this. I'd keep this flexibility of turning info into hashes in your back pocket!